packages/asc-ui/src/components/Input/Input.stories.tsx

```diff
@@ -44,9 +44,15 @@ export const InputWithRef = () => {
       console.log('Value changed!', (event.target as HTMLInputElement).value)
     }
 
```

Comment:
We can just return if the element does not exist instead, this will prevent the listener from being added.

```ts
if (!element) {
  return undefined
}
```